presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -20,15 +20,23 @@ import {
     computeRate,
     formatCount,
     formatDataSize,
-    formatDataSizeBytes, formatDuration, formatShortDateTime,
```

Comment:
Looks like the earlier comments about arbitrary were addressed here ... might want to move to the commits that introduce them